---
int32 motion_phase_count